CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+
+## Notes
+
+- **Before spending lots of time on something, ask for feedback on your idea first!**
+- Please search issues and pull requests before adding something new to avoid duplicating efforts and conversations.
+- Issues which are considered to be easy to solve are marked with the label `easy`: [Browse all easy issues](https://github.com/LibrePCB/LibrePCB/labels/easy).
+
+## Getting Started
+
+- Make sure you have a [GitHub account](https://github.com/signup/free).
+- Open a new issue for your idea, assuming one does not already exist.
+- Fork the repository on GitHub.
+- Have a look at our [development resources](https://github.com/LibrePCB/LibrePCB/tree/master/dev), 
+  especially at the [Doxygen documentation](http://librepcb.org/LibrePCB-Doxygen/master/).
+- When using QtCreator, import and use our [code style guide file](https://github.com/LibrePCB/LibrePCB/blob/master/dev/CodingStyle_QtCreator.xml).
+
+## Making Changes
+
+- Create a topic branch from where you want to base your work.
+  - This is usually the master branch.
```

Comment:
I'd remove this line. It adds no useful information, the next line already mentions master.